pkg/overlay/cache.go

```diff
@@ -399,15 +399,39 @@ func (cache *Cache) GetMissingPieces(ctx context.Context, pieces []*pb.RemotePie
 	return missingPieces, nil
 }
 
-func getIP(ctx context.Context, target string) (_ string, err error) {
+func getIP(ctx context.Context, target string) (ip net.IPAddr, err error) {
```

Comment:
the `ip` return parameter is declared and never used, it was not needed to be changed from `_`